tests/test_build_stub.nf.test

```diff
@@ -0,0 +1,41 @@
+nextflow_pipeline {
+
+    name "Test pipeline"
+    script "../main.nf"
+    profile "test_build"
+    options "-stub" // TODO remove once the pipeline is fixed
+    tag "pipeline"
+    tag "pipeline_rnafusion"
+    tag "test_build_stub"
+
+    test("stub test_build") {
+
+        when {
+            params {
+                outdir          = "$outputDir"
+                cosmic_username = "user" // placeholder string to avoid null input val in process
+                cosmic_passwd   = "password" // placeholder string to avoid null input val in process
+                all             = true
+            }
+        }
+
+        then {
+            // stable_name: All files + folders in ${params.outdir}/ with a stable name
+            // Ignore files with timestamps in their names
+            def stable_name = getAllFilesFromDir(params.outdir, relative: true, includeDir: true, ignore: ['pipeline_info/*.{html,json,txt}'])
+            // stable_path: All files in ${params.outdir}/ with stable content
+            def stable_path = getAllFilesFromDir(params.outdir, ignoreFile: 'tests/.nftignore')
+            assertAll(
+                { assert workflow.success},
+                { assert snapshot(
+                    // Number of successful tasks
+                    workflow.trace.succeeded().size(),
+                    // All stable path name, with a relative path
+                    stable_name,
+                    // All files with stable contents
+                    stable_path
```

Comment:
Consider removing

Reply:
In all honesty, stub tests achieve nothing, I added them more as a placeholder for whenever we are able to run the actual test profiles. Ill add a TODO to it, but I prefer to leave it like this